lms/resources/_js_config.py

```diff
@@ -392,5 +392,10 @@ def _sync_api(self):
                     "context_id": req.params["context_id"],
                     "custom_canvas_course_id": req.params["custom_canvas_course_id"],
                 },
+                "group_info": {
+                    key: value
+                    for key, value in req.params.items()
+                    if key in GroupInfo.columns()
+                },
```

Comment:
I changed this to only add into `group_info` that request params that're relevant to the `group_info` table by filtering based on `GroupInfo.columns()`. Previously it was adding _all_ the request params here, including things like the OAuth 1 signature